.circleci/config.yml

```diff
@@ -123,10 +123,15 @@ commands:
       - restore_cache:
           keys:
             - v1-dependencies-{{ checksum "./tmp/checksum.txt" }}
+
+  save-gradle-cache:
+    steps:
       - run:
-          name: Download dependencies
-          command: ./gradlew dependencies
+          name: Generate cache key
+          command: ./scripts/checksum.sh ./tmp/checksum.txt
```

Comment:
will it re-generate a different checksum  from the checksum.txt in the restore-gradle-cache job?

Reply:
they will be the same. `checksum.txt` depends on all `build.gradle.kts` + `Project.kt`
Leave it here for more consistency.